src/index.js

```diff
@@ -31,7 +35,14 @@ function startElectron (config, ports) {
         args = debugPortsArgs.concat(extraArgs);
     }
 
-    spawn(cmd, args, { stdio: 'ignore' });
+    var proc = spawn(cmd, args, { stdio: ['ignore', 'pipe', 'pipe'] });
+
+    proc.stdout.on('data', (buf) => {
```

Comment:
The TestCafe codestyle prefers writing arrow functions in a single line when it's possible. The data handlers are quite short, so we can rewrite them as one-liners.

Reply:
willfix.